pkg/controller/spd/indicator-plugin/manager.go

```diff
@@ -39,7 +40,7 @@ type IndicatorUpdater interface {
 	UpdateExtendedIndicatorSpec(_ types.NamespacedName, _ []apiworkload.ServiceExtendedIndicatorSpec)
 	UpdateBusinessIndicatorSpec(_ types.NamespacedName, _ []apiworkload.ServiceBusinessIndicatorSpec)
 	UpdateSystemIndicatorSpec(_ types.NamespacedName, _ []apiworkload.ServiceSystemIndicatorSpec)
-	UpdateBusinessIndicatorStatus(_ types.NamespacedName, _ []apiworkload.ServiceBusinessIndicatorStatus)
+	UpdateBusinessIndicatorStatus(_ types.NamespacedName, _ []apiworkload.ServiceBusinessIndicatorStatus, _ []apiworkload.AggPodMetrics)
```

Comment:
the logic of update AggMetrics can use an independent function`UpdateAggMetrics(_ types.NamespacedName, _ []apiworkload.AggPodMetrics)` like [UpdateAggMetrics](https://github.com/tibberzh/katalyst-core/blob/496361573af6d697754b9ffa0b555206bdcc420c/pkg/controller/spd/indicator-plugin/manager.go#L43) in #589

Reply:
I didn't notice there's another open mr working on the same thing, so was thinking just leverage the existing status function. Just curious if there're other consideration to make it separate?

Reply:
it's beter to separate the function of UpdateAggMetrics and UpdateBusinessIndicatorStatus, because all the update function like `UpdateExtendedIndicatorSpec`, `UpdateBusinessIndicatorSpec` update different aspects of SPD